src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -345,7 +345,7 @@ public void run() {
         values.add(new StringParameterValue("ghprbPullAuthorLoginMention", "@" + cause.getPullRequestAuthor().getLogin()));
         
         values.add(new StringParameterValue("ghprbPullDescription", escapeText(String.valueOf(cause.getShortDescription()))));
-        values.add(new StringParameterValue("ghprbPullTitle", String.valueOf(cause.getTitle())));
+        values.add(new StringParameterValue("ghprbPullTitle", escapeText(String.valueOf(cause.getTitle()))));
         values.add(new StringParameterValue("ghprbPullLink", String.valueOf(cause.getUrl())));
         values.add(new StringParameterValue("ghprbPullLongDescription", escapeText(String.valueOf(cause.getDescription()))));
```

Comment:
i think the same thing needs to be done here. my descriptions often contain ` `` ` escaped quotes which cause shell failures.